docs/getting-started/configuration.md

```diff
@@ -191,7 +191,7 @@ components: {
 That should be it. You can stop your Backstage App. When you start it again and
 go to your Backstage portal in your browser, you should have your login prompt!
 
-To learn more about Authentication in Backstage, there's the following docs you
+To learn more about Authentication in Backstage, here are the following docs you
```

Comment:
```suggestion
To learn more about Authentication in Backstage, here are some docs you
```

the original was kinda correct but using language a bit loosely - the fixed version introduced another oddity though